### PR TITLE
CWP: fix public draft2 URL after archival

### PR DESCRIPTION
**Pull request by jouvin**
- Draft2 URL also removed from main page

### CODE REVIEW

File: _activities/cwp.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ we are reaching out to other HEP experiments around the world to participate.
 ## Final Roadmap Paper
 
 The second draft of the CWP, the 
-[Roadmap for HEP Software and Computing R&D for the 2020s](https://docs.google.com/document/d/1mXJ51Xi1FEcQKzf6jJknmkC3MGeZqjPjgtoj3pVD6rk){:target="_cwp_roadmap"} 
+[Roadmap for HEP Software and Computing R&D for the 2020s](https://docs.google.com/document/d/1RIcnj7DBNOoQ1DT45WCGFboS0tCDeJcvAi2xjPv_UVQ){:target="_cwp_roadmap"} 
 report, has been released to the community for comment (also available as a 
 [pdf]({{site.docs.url}}/CWP/papers/roadmap/HSF-Community-White-Paper-v0.2.pdf)). You are **encouraged to read it** and
 **provide feedback**. The document will remain open for comments **until December 1**.
```
Comment (graeme-a-stewart) [inline _activities/cwp.md]: This text is out of date. Can we update it to "Comments were received until 4 December and the final paper is now being prepared."

Comment (graeme-a-stewart) [reply _activities/cwp.md]: Also, remove the next paragraph that describes the mechanics of commenting.

**Comment by jouvin** [comment]
Yes sure. Sorry I checked the text on the main page but on the CWP one!

**Comment by jouvin** [comment]
Should be ok.

**Pull request merged**